api/app/src/test/kotlin/packit/integration/repository/PacketRepositoryTest.kt

```diff
@@ -1,29 +1,24 @@
-package packit.unit.repository
+package packit.integration.repository
```

Comment:
I deleted the repositorry unit tests which just duplicated the tests in the integration tests but with a fake db.